app/grants/models.py

```diff
@@ -370,8 +370,8 @@ def get_is_ready_to_be_processed_from_db(self):
         if not self.subscription_contribution.exists():
             return True
         last_contribution = self.subscription_contribution.order_by('created_on').last()
-        period = last_contribution.created_on + timedelta(0, round(self.real_period_seconds))
```

Comment:
Somone must have changes this underneath me. I assume both work. is there a preferred implementation

Reply:
@captnseagraves I did, as mentioned on slack.

```
period = last_contribution.created_on + timedelta(0, round(self.real_period_seconds))
return period > timezone.now()
```

This avoids the need to force `period` to `float` and each datetime to timestamp.  We can perform the same checks against the datetime object itself without the need for this.
Also, in the changes proposed here, you've flipped the operator.  This will now return `True` when last contribution + period is less than now.

Reply:
@mbeacom cool. if we dont need to force the type changes that if great. However, as I look at this I realize we can just use `next_contribution_date`, which i can calculate as `self.next_contribution_date = timezone.now() + timedelta(0, round(self.real_period_seconds))` in the `successful_contribution` function. We do want to flip the operator though. We want to know when the `next_contributoin_date` is less than now. This tells us that the subscription is now ready to be processed.